src/Misc/DictionaryOperations.jl

```diff
@@ -0,0 +1,89 @@
+#=
+ MIT License
+
+ Copyright (c) 2010-present David A. Kopriva and other contributors: AUTHORS.md
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ --- End License
+=#
+
+#=
+    Some useful getters for a dictionary
+=#
+arrayRegex = r"(?<=\[).+?(?=\])"
```

Comment:
```suggestion
const arrayRegex = r"(?<=\[).+?(?=\])"
```
For type stability when using global variables

Reply:
Okay, is this something I should always keep in mind for globals?

Reply:
Yes

Reply:
https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-global-variables